### PR TITLE
Add 'Score quiz' functionality to Result

### PR DESCRIPTION
**Pull request by andreiuta97**
Add 'Score quiz' functionality.
Now, an Admin can score a quiz.
PHPTRAININ-399: https://jira.evozon.com/browse/PHPTRAININ-399

### CODE REVIEW

File: src/Controller/ResultController.php [hunk 1]
```diff
@@ -98,4 +102,23 @@ public function getResult(Request $request, array $requestAttributes): Response
...
+     * @return Response
+     */
+    public function scoreResult(Request $request, array $requestAttributes): Response
+    {
+        $quizInstanceId = $requestAttributes[self::QUIZ_INSTANCE_ID];
+        $score = $request->getParameter('score');
+        $this->quizInstanceService->saveScore($quizInstanceId, $score);
+        $body = Stream::createFromString('');
+        $response = new Response($body, '1.1', 301, '');
+        $response = $response->withHeader('Location', '/listResults');
```
Comment (Raresmldvn) [inline src/Controller/ResultController.php:120]: I will forever insist on creating a separate redirect method to use somewhere :D

File: src/Entity/QuizInstance.php [hunk 2]
```diff
@@ -96,11 +96,11 @@ public function setName(string $name)
     }
 
     /**
-     * @param string $type
+     * @param string $description
      */
-    public function setType(string $type)
+    public function setDescription(string $description)
```
Comment (Raresmldvn) [inline src/Entity/QuizInstance.php]: return type

File: src/Controller/ResultController.php (hunk 1 above)
Comment (andreiuta97) [reply src/Controller/ResultController.php:120]: Did it. There is a separate PR for this :D

File: src/Entity/QuizInstance.php (hunk 2 above)
Comment (andreiuta97) [reply src/Entity/QuizInstance.php]: Resolved in one of the 'search' PR you merged today.

**Pull request merged**